pytpcc/tpcc.py

```diff
@@ -53,6 +54,24 @@
     '%(asctime)s [%(funcName)s:%(lineno)03d] %(levelname)-5s: %(message)s'))
 logging.getLogger('').addHandler(console)
 
+NOTIFY_PHASE_START_PATH = '/data/workdir/src/flamegraph/notify_phase_start.py'
```

Comment:
Why is the notify command in the flamegraph repo?

Reply:
The DSI repo would be the natural choice here, but it is not rsynced over to the workload client. The flamegraph repo is the most profiling related repo I could find and is a repo where the script is "out of the way".

From [this PR](https://github.com/mongodb-forks/flamegraph/pull/1):

### Why are the scripts in this repo?
These scripts would semantically make the most sense in the DSI repo. However, when running DSI locally, a) the DSI folder is not rsynced to the workload client, and b) the DSI folder on the local machine is in the directory above the work directory, while on Evergreen it's located in src/dsi. See [this Slack discussion](https://mongodb.slack.com/archives/C044WBW0189/p1704728988752149) for more details.